README.md

```diff
@@ -50,6 +50,12 @@ There are methods for the four canonical event types: `created`, `updated`,
 the publisher floods with `noop`s for all existing entities so subscribers can
 refresh their view of the domain.
 
+A timestamp argument may be passed (it will be set by the bus automatically
+otherwise); it must be an integer number of milliseconds since the UNIX Epoch:
+
+```ruby
+client.created('widgets', 'https://app.example.com/widgets/1', 1473080555409)
```

Comment:
> One future improvement could be for Routemaster to prevent future timestamps to be set (either publisher- or bus-side).

This was my main concern, looks like a sensible _feature_.